osutil/disks/mockdisk.go

```diff
@@ -93,6 +93,21 @@ func (d *MockDiskMapping) FindMatchingPartitionWithPartLabel(label string) (Part
 	}
 }
 
+func (d *MockDiskMapping) FindMatchingPartitionWithNode(node string) (Partition, error) {
```

Comment:
Is this needed by this PR?

Reply:
uhh no, must have missed that when stitching the patches

Reply:
I've force pushed a change dropping this.